pkg/graveler/graveler.go

```diff
@@ -1571,6 +1571,29 @@ func (g *KVGraveler) listStagingArea(ctx context.Context, b *Branch) (ValueItera
 	return NewCombinedIterator(itrs...), nil
```

Comment:
I think you can also replace the DB graveler call in `SetHooksHandler` and remove the DB graveler from here entirely.

Reply:
done